multi-node/aws/cmd/kube-aws/command_up.go

```diff
@@ -26,7 +26,7 @@ var (
 func init() {
 	cmdRoot.AddCommand(cmdUp)
 	cmdUp.Flags().BoolVar(&upOpts.export, "export", false, "Don't create cluster, instead export cloudformation stack file")
-	//	cmdUp.Flags().BoolVar(&upOpts.update, "update", false, "update existing cluster with new cloudformation stack")
+	cmdUp.Flags().BoolVar(&upOpts.update, "update", false, "update existing cluster with new cloudformation stack")
```

Comment:
Note that @cgag is talking about flag _descriptions_, not the flags themselves. 

Going lowercase sounds good to me and seems to be what a lot of other CLIs do.